.rultor.yml

```diff
@@ -15,4 +18,4 @@ release:
   script: bash .misc/release.sh
 
 decrypt:
-  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
+  rultor_secrets.sh: "repo/.misc/rultor_secrets.sh.asc"
```

Comment:
Hi,

it looks like you're missing a railing newline before the end of the file here. [This article](http://stackoverflow.com/a/5813359/3212182) has more information about this issue.